eval.go

```diff
@@ -76,13 +76,18 @@ func (e *SetExpr) eval(app *App, args []string) {
 	case "shell":
 		gOpts.shell = e.val
 	case "showinfo":
-		if e.val != "none" && e.val != "size" && e.val != "time" {
-			msg := "showinfo should either be 'none', 'size' or 'time'"
+		if e.val != "none" && e.val != "size" && e.val != "time" && e.val != "time:size" && e.val != "size:time" {
```

Comment:
@IvanMenshykov this is still not generic. We will need to check all permutations once we start adding different types which will quickly become infeasible. I think this check should best be combined with the for loop below. Also `none` is going to be irrelevant.